engines/dfc_provider/app/services/dfc_loader.rb

```diff
@@ -2,15 +2,31 @@
 
 class DfcLoader
   def self.connector
-    @connector ||= load_vocabularies
+    unless @connector
+      @connector = DataFoodConsortium::Connector::Connector.instance
+      load_context
+      load_vocabularies
+    end
+
+    @connector
+  end
+
+  def self.load_context
+    JSON::LD::Context.add_preloaded("http://www.datafoodconsortium.org/") {
+      JSON::LD::Context.parse(read_file("context_1.8.2")["@context"])
```

Comment:
All releases of the ontology now have a versioned context file associated with them. We should target latest release (currently v1.14.0, soon to be v.1.15.0 for Variants)